Sources/DatadogSDKTesting/DDTestSession.swift

```diff
@@ -0,0 +1,244 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2020-2021 Datadog, Inc.
+ */
+
+import Foundation
+@_implementationOnly import OpenTelemetryApi
+@_implementationOnly import OpenTelemetrySdk
+@_implementationOnly import SigmaSwiftStatistics
+
+public class DDTestSession: NSObject {
+    var bundleName = ""
+    var bundleFunctionInfo = FunctionMap()
+    var codeOwners: CodeOwners?
+    var testFramework = "Swift API"
+
+    public init(bundleName: String) {
```

Comment:
As it is a `public` API, does it make sense / is it safe for the user to create several instances of `DDTestSession`?

We might have a slightly better control over this (today and in the future), if using dedicated initializer:
```swift
public class DDTestSession: NSObject {
   public static func initialize(bundleName: String) -> DDTestSession {
      return DDTestSession(bundleName: bundleName)
   }

   internal init(bundleName: String) { /* ... */ }
}
```
E.g. one day we may want to handle bad usage:
```swift
public static func initialize(bundleName: String) -> DDTestSession {
   guard !alreadyInitialized(with: bundleName) else {
      print("""
      `DDTestSession` was already created for \(bundleName). Make sure you initialize it only one for given bundle name.
      Returning no-op instance.
      """)
      return NoOpDDTestSession()
   }
   return DDTestSession(bundleName: bundleName)
}
```